src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -76,10 +76,13 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
 
   const userDisplayName: string = user?.partyName ?? DEFAULT_PARTY_NAME;
 
-  const containerClasses = classNames("UserAvatar", containerClassName, {
-    "UserAvatar--clickable": onClick !== undefined,
-    [`UserAvatar--${size}`]: size,
-  });
+  const sizeStyleName = `avatarContainer__${size}`;
+
+  const containerClasses = classNames(
+    styles.avatarContainer,
+    size && styles[sizeStyleName],
+    containerClassName
+  );
```

Comment:
`undefined` is ignored by `classnames` library.

```suggestion
  const containerClasses = classNames(
    styles.avatarContainer,
    styles[size],
    containerClassName
  );
```

Reply:
Yeah, but, Typescript gets very cross about trying to use `undefined` as a key